### PR TITLE
Added Documentation for Binary Search For Repeated Elements

### LINKED ISSUE #8121: [Create New Issue]: Binary Search for repeated elements
### Description

I will be demonstrating and implementing the code for Binary Search Algorithm for repetitive elements

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by VaibhavMalik4187**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #8121

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [X] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [X] C/CPP

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by VaibhavMalik4187** [comment]
@Saviour1001 @Suvraneel Please review my PR

**Comment by Suvraneel** [comment]
@gunjan-g

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Please remove this hash sign from C/CPP

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Add your file link into respective folder's readme.md

**Comment by vasu-1** [comment]
Rest of the things LGTM 🌝

**Comment by VaibhavMalik4187** [comment]
> 
> 
> Please remove this hash sign from C/CPP

Done

**Comment by VaibhavMalik4187** [comment]
> Add your file link into respective folder's readme.md
Done

**Comment by VaibhavMalik4187** [comment]
@vasu-1 I have made the changes that you requested, now kindly review again and merge my PR

File: C_CPP/Algorithmic Approaches/Divide & Conquer/BinarySearchForRepeatedElements.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+# **Binary Search For Repeated Elements**
+
+<br>
+
```
Comment (Suvraneel) [inline C_CPP/Algorithmic Approaches/Divide & Conquer/BinarySearchForRepeatedElements.md:4]:
```suggestion
  
  
```
Remove all html tags... use 2 spaces at end of each line for line breaks

File: C_CPP/Algorithmic Approaches/Divide & Conquer/BinarySearchForRepeatedElements.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- The algorithm is a divide and conquer algorithm, at a time it divides the container into 2 halves out of which, one part is discarded straight away, and the  same process happens for the remaining part.
+- The worst case ***TIME COMPLEXITY*** of this algorithm is : ***O(log(Size of the Container))*** which is when, if the key is located at either of the ends
+
+<br>
+
+## **Code**
+
+<br>
+
+```C++
```
Comment (Suvraneel) [inline C_CPP/Algorithmic Approaches/Divide & Conquer/BinarySearchForRepeatedElements.md:78]:
```suggestion
  

```C++
```

**Review by Suvraneel — CHANGES_REQUESTED** [review]
@VaibhavMalik4187

**Comment by VaibhavMalik4187** [comment]
@Suvraneel I have removed all the <br> tags, can you please review my PR again?

**Comment by Suvraneel** [comment]
@VaibhavMalik4187  LGTM 🚀 :octocat:

**Pull request merged**